### PR TITLE
Format date fields in CSV/Shapefile exports

### PR DESCRIPTION
**Pull request by steventlamb**
connects to #1590 on github

### CODE REVIEW

File: src/nyc_trees/apps/census_admin/tasks.py [hunk 1]
```diff
@@ -27,6 +29,15 @@
 from apps.survey.models import Blockface
 from apps.event.models import Event
 
+_DATETIME_FIELDS = ('created_at',
```
Comment (jwalgran) [inline src/nyc_trees/apps/census_admin/tasks.py]: This list omits some fields (``last_login``, ``date_joined``, maybe more). Is walking the model metadata to find datetime fields an option?

File: src/nyc_trees/apps/census_admin/tasks.py [hunk 2]
```diff
@@ -27,6 +29,15 @@
...
+                    'updated_at',
+                    'territory_updated_at',
+                    'canceled_at',
+                    'expries_at',
```
Comment (maurizi) [inline src/nyc_trees/apps/census_admin/tasks.py]: typo, `expries_at` should be `expires_at`

File: src/nyc_trees/apps/census_admin/tasks.py [hunk 3]
```diff
@@ -27,6 +29,15 @@
...
+                    'reminder_sent_at',
+                    'begins_at',
+                    'ends_at')
```
Comment (maurizi) [inline src/nyc_trees/apps/census_admin/tasks.py]: some more fields: `'date_joined', 'last_login'`

**Comment by jwalgran** [comment]
The formatter is working correctly for CSVs and shapefiles.

**Comment by steventlamb** [comment]
@jwalgran @maurizi comments addressed

**Comment by jwalgran** [comment]
+1 I am seeing EST dates  in YYYY-MM-DD HH:MM:SS format in all the dumped files

**Pull request merged**